www/lib/CordovaAppLoader.js

```diff
@@ -172,8 +172,11 @@ var CordovaAppLoader =
 	            self.corruptNewManifest = true;
 	            resolve(null);
 	          }
-	          // Yes, we've updated and we've succeeded.
-	          resolve(false);
+		      else
```

Comment:
I have OCD about these issues as well...this is what I see on my machine:

![image](https://cloud.githubusercontent.com/assets/59510/7909167/0513ecc2-0853-11e5-8e5d-501d12db8313.png)